### PR TITLE
Customize buttons to be TW blue!

### PR DESCRIPTION
**Pull request by taylortrimble**
Been waiting for a good time to do this

### CODE REVIEW

File: static/css/teslaworks.css [hunk 1]
```diff
@@ -17,6 +17,47 @@ a:hover, a:focus {
...
+.btn-primary.disabled:active,
+.btn-primary[disabled]:active,
+fieldset[disabled] .btn-primary:active,
+.btn-primary.disabled.active,
+.btn-primary[disabled].active,
+fieldset[disabled] .btn-primary.active {
+  background-color: #1a80f7;
+  border-color: #197aeb;
+}
+
```
Comment (mplewis) [inline static/css/teslaworks.css:60]: This is massive, nice work!

**Comment by mplewis** [comment]
Ya like blue buttons better than green? I dig. :sailboat:

**Comment by taylortrimble** [comment]
Yeah! Something is telling me I need to save those green buttons for something. Something "GO!" that isn't on the page every time.

**Comment by taylortrimble** [comment]
What do you think about making the Start one green again though?

I also plan on having a collapsible form come out under "join this project" that could have a green button appear. (Already glad to have saved it now! :smile:)

**Comment by mplewis** [comment]
Can we make the start button bigger and centered like the rest of the form?

**Comment by taylortrimble** [comment]
Interesting: I just tried most combinations, and the best looking is just a wider left-justified button. With a narrower form it goes back to… looking best just leaving it. You okay with that?

**Pull request merged**